### ISSUE TITLE
Presentation 5 - must-haves

### ISSUE THREAD

**Issue by j-ti** [j-ti/SES-House#66]
• the selected uncontrollable resource and load
• the datasets; how each dataset is partitioned into training and test datasets
• the network architectures, i.e. the number of layers and the number of neurons at each layer for each forecasting task
• selected loss function of each training task
• selected activation functions of each training task
• selected optimizer of each training task
